### PR TITLE
Delegating construction to CommonHelper.getConstructor()

### PR DESCRIPTION
**Pull request by m-froehlich**
Utilizing CommonHelper.getConstructor() to construct instance of ConfigFactory.

### CODE REVIEW

File: pac4j-core/src/main/java/org/pac4j/core/config/ConfigBuilder.java [hunk 1]
```diff
@@ -19,14 +19,7 @@ public synchronized static Config build(final String factoryName, final Object..
...
 
-            var tccl = Thread.currentThread().getContextClassLoader();
-            final Class<ConfigFactory> clazz;
-            if (tccl == null) {
-                clazz = (Class<ConfigFactory>) Class.forName(factoryName);
-            } else {
-                clazz = (Class<ConfigFactory>) Class.forName(factoryName, true, tccl);
-            }
-            final var factory = clazz.getDeclaredConstructor().newInstance();
+            final var factory = (ConfigFactory) org.pac4j.core.util.CommonHelper.getConstructor( factoryName ).newInstance();
```
Comment (mmoayyed) [inline pac4j-core/src/main/java/org/pac4j/core/config/ConfigBuilder.java:22]: Is there a reason you're using the full package name here?

Comment (mmoayyed) [reply pac4j-core/src/main/java/org/pac4j/core/config/ConfigBuilder.java:22]:
```suggestion
            final var factory = (ConfigFactory) CommonHelper.getConstructor(factoryName).newInstance();
```

Comment (m-froehlich) [reply pac4j-core/src/main/java/org/pac4j/core/config/ConfigBuilder.java:22]: Not really. I wanted to keep my changes as locally as possible. Feel free to use an import.

Comment (leleuj) [reply pac4j-core/src/main/java/org/pac4j/core/config/ConfigBuilder.java:22]: OK. Will fix

**Pull request merged**